recipes/marvin/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "marvin" %}
+{% set version = "2.3.8" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f96628c5e7292ef02cf4bc1f4e6692df23cd1622e4f8c7cfcb2190bda38619d6
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=45
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >={{ python_min }}
+    - cachetools >=5
+    - fastapi
+    - httpx >=0.24.1
+    - jinja2 >=3.1.2
+    - jsonpatch >=1.33
+    - openai >=1.21.0
+    - prompt-toolkit >=3.0.33
+    - pydantic >=2.4.2
+    - pydantic-settings
+    - rich >=12
+    - tiktoken >=0.4.0
+    - typer >=0.9.0
+    - typing_extensions >=4.0.0
+    - tzdata >=2023.3
```

Comment:
I think this needs to be `python-tzdata` to get the python library rather than the C-library of the same name.